quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -199,22 +197,47 @@ impl Ingester {
         if queue_ids.is_empty() {
             return Ok(());
         }
-        info!("closing {} shard(s)", queue_ids.len());
+        info!("recovering {} shard(s)", queue_ids.len());
```

Comment:
I think the function itself could use a comment. I cannot add a CR comment on that but something like 
```
/// During the initialization of the ingester, we list all of the queues stored in
/// mrecordlog. Empty queues are simply deleted, while queues that are not empty,
/// are recovered, but closed. 
///
/// Closed means they are in a read-only state: available for their
/// indexing to be resumed, but not available to receive more records.
```